src/adapters/jquery.js

```diff
@@ -1,14 +1,12 @@
-(function() {
   'use strict'
 
-  var $ = window.jQuery
-  var Waypoint = window.Waypoint
+  var jquery = global.jQuery  //require('jquery')
```

Comment:
I tried to require jquery, but jquery's npm package doesn't contain $.forEach utilities and stuff. So this comment is more redundant. I still can try to shim jquery import.